sdk/schemaregistry/azure-schemaregistry-avroencoder/README.md

```diff
@@ -44,71 +44,71 @@ pip install azure-identity
 pip install aiohttp
 ```
 
-**Create AvroSerializer using the azure-schemaregistry library:**
+**Create AvroEncoder using the azure-schemaregistry library:**
 
 ```python
 from azure.schemaregistry import SchemaRegistryClient
-from azure.schemaregistry.serializer.avroserializer import AvroSerializer
+from azure.schemaregistry.encoder.avroencoder import AvroEncoder
 from azure.identity import DefaultAzureCredential
 
 credential = DefaultAzureCredential()
 # Namespace should be similar to: '<your-eventhub-namespace>.servicebus.windows.net'
 fully_qualified_namespace = '<< FULLY QUALIFIED NAMESPACE OF THE SCHEMA REGISTRY >>'
 group_name = '<< GROUP NAME OF THE SCHEMA >>'
 schema_registry_client = SchemaRegistryClient(fully_qualified_namespace, credential)
-serializer = AvroSerializer(client=schema_registry_client, group_name=group_name)
+encoder = AvroEncoder(client=schema_registry_client, group_name=group_name)
 ```
 
 ## Key concepts
 
-### AvroSerializer
+### AvroEncoder
 
-Provides API to serialize to and deserialize from Avro Binary Encoding plus a
-header with schema ID. Uses [SchemaRegistryClient][schemaregistry_client] to get schema IDs from schema content or vice versa.
+Provides API to encode to and decode from Avro Binary Encoding plus a
+content type with schema ID. Uses [SchemaRegistryClient][schemaregistry_client] to get schema IDs from schema content or vice versa.
 
-### Message format
-
-The same format is used by schema registry serializers across Azure SDK languages.
-
-Messages are encoded as follows:
+### Supported message models
 
-- 4 bytes: Format Indicator
+Support has been added to certain Azure Messaging SDK model classes for interoperability with the `AvroEncoder`. These models are subtypes of the `MessageType` protocol defined under the `azure.schemaregistry.encoder.avroencoder` namespace. Currently, the supported model classes are:
```

Comment:
nit: I feel we could copy paste the protocol definition code into this section?

Reply:
will fix in separate docs PR